src/commands/configure_maven.yml

```
@@ -0,0 +1,21 @@
+description:
+  Generate temporary Cloudsmith credentials using OIDC and construct a maven registry URL string for a specific Cloudsmith repository. Checks for existence of .m2 and settings.xml maven configuration files and creates them with Cloudsmith authentication specifics if they do not exist.
+parameters:
+  repository:
+    description: The identity/slug of the Cloudsmith repository
+    type: string
+steps:
+  - run:
+      name: Cloudsmith - Configure defaults
+      command: <<include(scripts/configure_cloudsmith_defaults.sh)>>
+  - cloudsmith-oidc/authenticate_with_oidc
+  - run:
+      name: Cloudsmith - Set environment variables
+      command: <<include(scripts/set_env_vars_for_maven.sh)>>
+      environment:
+        CLOUDSMITH_REPOSITORY: <<parameters.repository>>
```

```suggestion
  - run: 
        command: set_env_vars_for_maven
        repository: <<parameters.repository>>
  
```
We could reuse the other command here